cmd/kured/main.go

```diff
@@ -804,6 +824,21 @@ func rebootAsRequired(nodeID string, booter reboot.Reboot, sentinelCommand []str
 	}
 }
 
+func lastSuccessfulRebootWithinMinRebootPeriod(node *v1.Node) bool {
+	if minRebootPeriod == 0 {
+		return false
+	}
+	if v, ok := node.GetAnnotations()[KuredLastSuccessfulRebootAnnotation]; ok {
```

Comment:
just use `node.Annotations` as on the other places and add a or check to the first if with `!annotateNodes` to also return false in this case.